server/server.go

```diff
@@ -66,6 +67,8 @@ type Server struct {
 	pullOperationsInProgress map[pullArguments]*pullOperation
 	// pullOperationsLock is used to synchronize pull operations.
 	pullOperationsLock sync.Mutex
+
+	*resourcecache.ResourceCache
```

Comment:
With the naming changes I suggested above, surely it needs to have this one as a named member instead of embedding...